client/api/openapi.yaml

```diff
@@ -78,6 +78,16 @@ paths:
         schema:
           type: string
         style: simple
+      - description: Optional flag to indicate if the file is incoming or outgoing.
+          SenderSupplied (tag 1500) is not required for incoming files.
+        explode: true
+        in: query
+        name: isIncoming
```

Comment:
Do we want to take this chance for a better name? Might we need to make other changes to files/validation on incoming files in the future?

Reply:
I debated a couple/few approaches:
1. Expose the ValidationOpts struct in the API models, and add a `SenderSuppliedOptional` option
    * This seems like the best long term solution, but would be a larger, and potentially breaking, change 
2. Add a query param specifically for SenderSupplied
3. Keep `isIncoming`

I ended up with option 3 because it keeps the changes minimal and makes it easy to add additional behaviors for `isIncoming` in the future. So far, SenderSupplied` is the only tag we've encountered that's different for incoming vs outgoing, but I'm making an assumption it won't be the last.

That said, if you think this approach takes the API down the wrong path I can come up with a different option.